apps/code-examples/src/app/code-examples/indicators/status-indicator/basic/status-indicator-demo.component.html

```diff
@@ -24,5 +24,8 @@
 
 <sky-status-indicator descriptionType="error" indicatorType="danger">
   Status indicator with help
-  <sky-help-inline class="sky-control-help"></sky-help-inline>
```

Comment:
same here, the pattern we've been following has been to have a second code example for inline help